conflowgen/api/container_dwell_time_distribution_manager.py

```diff
@@ -58,3 +63,39 @@ def set_container_dwell_time_distribution(
             sanitized_distribution
         )
         DataSummariesCache.reset_cache()
+
+    def get_average_container_dwell_time(self, start_date, end_date) -> float:
```

Comment:
Typically, the idea of the api submodule is that no logic computations are achieved here, this is only the interface and the input variables are just forwarded. Could the computatoin be moved to a new service that just covers this?

In addition, please add typehints for start_date and end_date.

Reply:
See [Create AverageContainerDwellTimeCalculatorService](https://github.com/1kastner/conflowgen/pull/187/commits/0038e76b8874ea6f01cb42ef974316d41d7d3aa1)